2.7.0/done/result_inductor.md

```diff
@@ -0,0 +1,458 @@
+## inductor
+### bc breaking
```

Comment:
For bc breaking and deprecation, can you include a 2.6.0 vs 2.7.0 before and after? Please check the guidance doc in the instructions for what is expected for these sections